fastboot-lt-2-10/instance-initializers/fastboot/head.js

```diff
@@ -15,5 +15,5 @@ export function initialize(instance) {
 
 export default {
   name: 'head-fastboot',
-  initialize: initialize
```

Comment:
Just had a look at it again... I *think* this is not needed, as this is not part of `app`
* for apps that are not using the new FastBoot build (beta FastBoot or not FB at all), this is merged into app through `filterInitializers` (https://github.com/ronco/ember-cli-head/pull/21/files#diff-168726dbe96b3ce427e7fedce31bb0bcR23), so the browser build should not include this
* for apps having the new FastBoot, this is only added to `treeForFastboot`

And the ember 2.4 test scenario seems to be ok!